packages/server/utils/setTierForOrgUsers.ts

```diff
@@ -0,0 +1,22 @@
+import getRethink from '../database/rethinkDriver'
+import {TierEnum} from 'parabol-client/types/graphql'
+
+const setTierForOrgUsers = async (orgId: string) => {
+  const r = await getRethink()
+  await r
+    .table('OrganizationUser')
+    .getAll(orgId, {index: 'orgId'})
+    .filter({removedAt: null})
```

Comment:
We only need to update on `OrganizationUser` objects with `removedAt = null`. Why? `OrganizationUser` row represents one instance of a "membership joining". Once `removedAt` field is populated, that `OrganizationUser` row will not be used again. In case the user is re-invited and rejoins, a new `OrganizationUser` row will be created.

Reply:
I think this knowledge deserves to be preserved as a multi-line comment, because it would be non-obvious to somebody looking at this code.